src/test/java/org/folio/utils/CalculationUtilsTest.java

```diff
@@ -28,28 +28,28 @@ void overExpendedShouldBeGreaterThenZeroWhenSumOfExpendedAwaitingPaymentGreaterT
 
   @ParameterizedTest
   @CsvSource({
-    "1800, 200, 1500, 100, 700, 200, 300",
```

Comment:
overEncumbrance should not change in any of these cases

Reply:
I was thinking it should not change assuming credited=0, but you added some credits with the first PR for MODFISTO-481, so I guess results need to be adjusted. Actually, to avoid having to re-check correctness for all over encumbrance values, we should use the values before MODFISTO-481, and only add credits so that the old expended is the new expended - credited. For instance if expended was 200 we can have expended=200, credited=0 or expended=300, credited=100.

Reply:
good point. Then I reverted back overEncumbrnace values and only changes are made for credited and expended amounts